api/shards.ts

```diff
@@ -0,0 +1,60 @@
+import got from '../libs/got'
+import { millify, version, versionColor } from '../libs/utils'
+import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+
+const SHARDS_REPO_URL = 'https://shardbox.org/'
+
+const client = got.extend({ prefixUrl: SHARDS_REPO_URL })
+
+export default createBadgenHandler({
+  title: 'Crystal shards',
+  examples: {
+    '/shards/v/kemal': 'version',
+    '/shards/v/crinja': 'version',
```

Comment:
Is it necessary to have 2 examples of the same badge topic?

Reply:
Absolutely not but it was modeled after https://badgen.net/npm :shrug:

Reply:
That's unnecessary, we need to update that 😅

Reply:
When I contributed my first badge I used `npm` as a sort of template. I always thought that the duplicated version badge is added intentionally to show that its color is calculated from the version itself. So I usually picked one pre v1 and one post v1 example to follow the suit. Reverting all that is out of scope for this PR but for starters, I'm gonna remove `crinja` because all (I guess :thinking:) shards are pre v1 anyway.